ocaml/xapi/pool_periodic_update_sync.mli

```diff
@@ -0,0 +1,40 @@
+(*
+ * Copyright (C) 2006-2009 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+val set_enabled : __context:Context.t -> value:bool -> unit
+
+(* Below exposed only for ease of testing *)
+
+val next_scheduled_datetime :
+     delay:float
+  -> utc_now:Ptime.t
+  -> tz_offset_s:int
+  -> int * int * int * int * int * int
+
+val utc_start_of_next_scheduled_day :
+     utc_now:Ptime.t
+  -> tz_offset_s:int
+  -> frequency:[< `daily | `weekly]
+  -> day_configed_int:int
+  -> Ptime.t
+
+val calc_delay :
+     utc_now:Ptime.t
+  -> utc_start_of_next_sched_day:Ptime.t
+  -> seconds_in_a_day:float
```

Comment:
This looks like a strange parameter. The seconds in a day (or seconds per day) is a constant. Is this the seconds into the day?